mne/epochs.py

```diff
@@ -498,15 +504,31 @@ def plot(self, epoch_idx=None, picks=None, scalings=None,
             Whether to halt program execution until the figure is closed.
             Useful for rejecting bad trials on the fly by clicking on a
             sub plot.
+        n_epochs : int
+            The number of epochs to show on a continuous plot. Defaults to 20.
+        n_channels : int
+            The number of channels per view on a continuous plot.
+            Defaults to 10.
+        fig_title : str | None
+            The title of the window. If None, epochs name will be displayed.
+            Defaults to None.
+        trellis : bool
+            Whether to use Trellis plotting. If False, the epochs are plotted
+            as continuous curves. Defaults to True.
```

Comment:
Yeah, that alternate phrasing sounds good too.

We'd probably get rid of this boolean parameter anyway once @jaeilepp goes for next PR since we'd have both in the same plot. And since it would be in the same release cycle, we don't need a deprecation. Is that correct @Eric89GXL ?